lib/Bindings/Python/dialects/om.py

```diff
@@ -38,11 +38,16 @@ def wrap_mlir_object(value):
   assert isinstance(value, BaseObject)
   return Object(value)
 
+def om_var_to_attribute(obj, none_on_fail: bool = False) -> ir.Attrbute:
+  if isinstance(obj, int):
```

Comment:
I think we'll need similar functionality for strings, bools, etc. right? That can come as follow ups.

Reply:
I think for bools we are still using `i1` , so `mlir::BoolAttr` should suffice. 
Seems like StringAttr` also doesn't require an explicit cast.
Will add lit tests for string and bool, and the API if required.